testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -2924,6 +2924,114 @@ public void testSparkAlterStructColumnType(StorageFormat storageFormat)
         onSpark().executeQuery("DROP TABLE " + sparkTableName);
     }
 
+    @Test(groups = {ICEBERG, PROFILE_SPECIFIC_TESTS})
+    public void testSparkReadingTrinoParquetBloomFilters()
+    {
+        String baseTableName = "test_spark_reading_trino_bloom_filters";
+        String trinoTableName = trinoTableName(baseTableName);
+        String sparkTableName = sparkTableName(baseTableName);
+
+        onTrino().executeQuery(
```

Comment:
@jkylling i've checked the following scenario (case sensitive column names)

```
spark
CREATE TABLE t1 (testInteger INTEGER, testLong BIGINT, testString STRING, testDouble DOUBLE, testFloat REAL)
USING iceberg
TBLPROPERTIES (
'write.parquet.bloom-filter-enabled.column.testInteger' = true, 
'write.parquet.bloom-filter-enabled.column.testLong' = true,
'write.parquet.bloom-filter-enabled.column.testString' = true,
'write.parquet.bloom-filter-enabled.column.testDouble' = true,
'write.parquet.bloom-filter-enabled.column.testFloat' = true
);

trino
INSERT INTO iceberg.default.t1
SELECT testInteger, testLong, testString, testDouble, testFloat 
FROM (
    VALUES
        (-999999, -999999, 'aaaaaaaaaaa', DOUBLE '-9999999999.99', REAL '-9999999.9999'),
        (3, 30, 'fdsvxxbv33cb', DOUBLE '97662.2', REAL '98862.2'),
        (5324, 2466, 'refgfdfrexx', DOUBLE '8796.1', REAL '-65496.1'),
        (999999, 9999999999999, 'zzzzzzzzzzz', DOUBLE '9999999999.99', REAL '-9999999.9999'),
        (9444, 4132455, 'ff34322vxff', DOUBLE '32137758.7892', REAL '9978.129887')
) AS DATA(testInteger, testLong, testString, testDouble, testFloat);

SELECT count(*) FROM iceberg.default.t1 WHERE testinteger in (-88777, 6711111);
```

```
queryStats

    "processedInputDataSize" : "25B",
    "failedProcessedInputDataSize" : "0B",
    "processedInputPositions" : 5,
```

While using lowercase column names however i see that the parquet bloom filter is effective
```
spark
CREATE TABLE t2 (testinteger INTEGER, testlong BIGINT, teststring STRING, testdouble DOUBLE, testfloat REAL)
USING iceberg
TBLPROPERTIES (
'write.parquet.bloom-filter-enabled.column.testInteger' = true, 
'write.parquet.bloom-filter-enabled.column.testLong' = true,
'write.parquet.bloom-filter-enabled.column.testString' = true,
'write.parquet.bloom-filter-enabled.column.testDouble' = true,
'write.parquet.bloom-filter-enabled.column.testFloat' = true
);
trino
INSERT INTO iceberg.default.t2
SELECT testInteger, testLong, testString, testDouble, testFloat 
FROM (
    VALUES
        (-999999, -999999, 'aaaaaaaaaaa', DOUBLE '-9999999999.99', REAL '-9999999.9999'),
        (3, 30, 'fdsvxxbv33cb', DOUBLE '97662.2', REAL '98862.2'),
        (5324, 2466, 'refgfdfrexx', DOUBLE '8796.1', REAL '-65496.1'),
        (999999, 9999999999999, 'zzzzzzzzzzz', DOUBLE '9999999999.99', REAL '-9999999.9999'),
        (9444, 4132455, 'ff34322vxff', DOUBLE '32137758.7892', REAL '9978.129887')
) AS DATA(testInteger, testLong, testString, testDouble, testFloat);
SELECT count(*) FROM iceberg.default.t2 WHERE testinteger in (-88777, 6711111);
```

```
queryStats
    "processedInputDataSize" : "0B",
    "failedProcessedInputDataSize" : "0B",
    "processedInputPositions" : 0,
```

Reply:
What would show create table be in Trino for the two tables?

Reply:
> What would show create table be in Trino for the two tables?

Lower-case column names should be fine IMO.

Reply:
@findinpath I've added `plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergParquetWithBloomFiltersMixedCase.java` to test Parquet files with Bloom filters on columns with mixed case. Please have a look and let me know what you think.